reference-tests/build.gradle

```diff
@@ -52,6 +58,11 @@ tasks.register('referenceBlockchainTests', Test) {
   description = 'Runs ETH reference blockchain tests.'
   dependsOn generateBlockchainReferenceTests
 
+  boolean isCiServer = System.getenv().containsKey("CI")
+  minHeapSize = isCiServer ? "8g" :"4g"
+  maxHeapSize = isCiServer ? "20g" : "8g"
+  jvmArgs '-Xmx512m', '-Xms128m'
```

Comment:
Wait, Xms and Xmx are used to set min and max heap size which are already set using minHeapSize and maxHeapSize. I am not sure which one will take effect.

Reply:
I've removed the xmx and xms args.